### PR TITLE
Updated nightmare bio lab

### PR DESCRIPTION
**Pull request by Akaineko-astasi**
* **Addressed Issue(s)**: 

#3785
* **Server Mode**: 

Renewal
* **Description of Pull Request**: 

* adds item drop to 2F (1st job) mobs, 
* adds item drop to 5F (3rd job) MVPs; 
* adds a number of warp points with F_Pieces_Of_Sentiment 
* adds a number of lore pieces (now when I'm reviewing them I suppose there has to be 5 phrases at each point, but I didn't notice it while collecting)

### CODE REVIEW

File: db/re/mob_db.txt [hunk 1]
```diff
@@ -680,12 +680,12 @@
...
+1634,SEYREN,Seyren,Seyren Windsor,142,194962,1,10386,7790,1,3505,1317,166,12,142,69,86,49,132,55,10,12,1,6,63,0x2003095,170,76,384,288,0,0,0,0,0,0,0,7345,1500,13030,1,12623,5,6223,1,6471,1,6469,200,13421,100,25129,1000,0,0,4358,1
+1635,EREMES,Eremes,Eremes Guile,140,190525,1,10329,6846,1,3055,1171,122,12,145,87,59,67,127,76,10,12,1,6,85,0x2003095,180,76,384,288,0,0,0,0,0,0,0,7347,2000,6223,1,2514,1,1262,3,1264,30,6471,2,1284,100,25129,1000,0,0,4360,1
+1636,HARWORD,Harword,Howard Alt-Eisen,142,348100,1,10854,7241,1,2762,1282,131,10,138,79,62,53,82,48,10,12,1,7,81,0x3095,180,76,384,288,0,0,0,0,0,0,0,7345,3000,6471,1,1311,200,12818,55,6469,100,2318,1,6223,10,25129,1000,0,0,4362,1
+1637,MAGALETA,Magaleta,Margaretha Sorin,140,220800,1,9895,7421,1,1277,3554,108,93,126,74,71,123,99,65,10,12,1,7,66,0x3295,180,1152,384,288,0,0,0,0,0,0,0,7347,1000,1647,200,6471,1,1561,5,2162,5,1659,10,6469,200,25129,1000,0,0,4364,1
 1638,SHECIL,Shecil,Cecil Damon,141,190255,1,10103,7577,14,3737,1248,76,15,121,126,67,80,165,42,10,12,1,7,64,0x3095,180,76,384,288,0,0,0,0,0,0,0,7345,1500,12014,55,18110,10,12623,3,6469,200,6471,1,1745,200,0,0,0,0,4368,1
```
Comment (Lemongrass3110) [inline db/re/mob_db.txt]:
```suggestion
1638,SHECIL,Shecil,Cecil Damon,141,190255,1,10103,7577,14,3737,1248,76,15,121,126,67,80,165,42,10,12,1,7,64,0x3095,180,76,384,288,0,0,0,0,0,0,0,7345,1500,12014,55,18110,10,12623,3,6469,200,6471,1,1745,200,25129,1000,0,0,4368,1
```

File: db/re/mob_db.txt [hunk 2]
```diff
@@ -2371,39 +2371,39 @@
...
+3223,V_B_HARWORD,Mechanic Harword,Mechanic Howard,187,18000000,1,3000000,3000000,1,10000,6000,400,200,320,160,400,120,260,100,10,12,1,7,81,0x6283695,100,76,384,288,0,0,0,0,0,0,0,23080,3000,20748,150,20748,500,22687,5000,6815,10000,6815,10000,6815,10000,0,0,0,0,4677,1
+3224,V_B_KATRINN,Warlock Katrinn,Warlock Kathryne,187,10800000,1,3000000,3000000,1,4000,12000,220,800,220,160,120,400,420,100,10,12,1,7,88,0x6283695,100,576,384,288,0,0,0,0,0,0,0,23016,3000,20748,150,20748,500,22687,5000,6817,10000,6817,10000,6817,10000,0,0,0,0,4678,1
+3225,V_B_SEYREN,Rune Knight Seyren,Rune Knight Seyren,189,14400000,1,3000000,3000000,1,9000,8000,800,400,340,200,400,100,400,120,10,12,1,7,83,0x6283695,100,76,384,288,0,0,0,0,0,0,0,23080,3000,20748,150,20748,500,22687,5000,6814,10000,6814,10000,6814,10000,0,0,0,0,4679,1
 3226,V_RANDEL,V Randel,Randel Lawrence,178,2550000,1,100000,50000,1,4000,2000,300,200,200,80,200,50,190,70,10,12,1,7,86,0x6203695,100,76,384,288,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,25131,1000,22687,500,23016,500,6814,500,19972,50,4690,1
```
Comment (Lemongrass3110) [inline db/re/mob_db.txt]: 3226-3232 also need an update I think

Comment (Akaineko-astasi) [reply db/re/mob_db.txt]: Absolutely, thanks for noticing!

**Comment by Lemongrass3110** [comment]
@Everade this is making things far better now than they currently are.
The rest can be fixed later on when we have better/clear sources and when we update the whole mob db.

**Pull request merged**